uiowa/uiowa
# [ITS] Alert notifications bug

Resolves https://github.com/uiowa/uiowa/issues/8106



# How to test

```
ddev blt ds --site=its.uiowa.edu  && ddev drush @its.local uli /admin/config/sitenow/its-settings
```
- Confirm "Secondary alert notification" field has email value
- Confirm validation on the secondary field won't allow multiple addresses
- Confirm validation on the other fields still works
- Go to `/node/add/alert` and add an alert, check "Send alert email on save" and save as "published"
- `ddev launch -m` and confirm that two separate emails were sent containing the same content
